le_utils/constants/languages.py

```diff
@@ -52,6 +54,53 @@ def _initialize_language_list():
 
 _LANGLOOKUP = {l.code: l for l in LANGUAGELIST}
 
-
 def getlang(code, default=None):
+    """
+    Try to lookup a Language object for `code` in internal representation defined
+    in resources/languagelookup.json.
+    Returns None if lookup by interner representation fails.
+    """
     return _LANGLOOKUP.get(code) or None
+    # should this be   return _LANGLOOKUP.get(code) or _LANGLOOKUP[default]  ???
+
+
+_LANGUAGE_NAME_LOOKUP = {l.name: l for l in LANGUAGELIST}
+
+new_items = {}
+for lang_name, lang_obj in _LANGUAGE_NAME_LOOKUP.items():
```

Comment:
Nice! Mind adding a small explanatory comment on what this does, e.g. `Add language names that are separated by semicolons, e.g. "Catalan; Valencian"`